backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -1162,7 +1162,6 @@ public void createWorkspaceRoleBinding(
       boolean orgScopedPublic,
       String globalSharing) {
     if (workspaceId != null && !workspaceId.isEmpty()) {
-      Role admin = getRoleByName(roleAdminName, null);
```

Comment:
this si just for cleanliness right? or was there a bug you fixed?

Reply:
it is just moved line where it is actually used. nothing else.